index.js

```diff
@@ -31,11 +33,28 @@ function runner (opts) {
 
   var server = http.createServer(function (req, res) {
 
-    if (/^\/bundle\.js/.test(req.url)) {
-      res.setHeader('content-type', 'application/javascript');
-      bundle.createReadStream().pipe(res);
-      return;
+    if (opts.input === 'javascript') {
+
+      if (/^\/bundle\.js/.test(req.url)) {
+        res.setHeader('content-type', 'application/javascript');
+        bundle.createReadStream().pipe(res);
+        return;
+      }
+
+      if (req.url == '/') {
+        fs.createReadStream(__dirname + '/static/index.html').pipe(res);
+        return;
+      }
+
+    } else if (opts.input === 'html') {
+
+      if (req.url == '/') {
+        bundle.createReadStream().pipe(injectScript(['/reporter.js'])).pipe(res);
```

Comment:
i got confused there for a second, since `bundle` can now also contain html we should rename it to `source`